website/pages/docs/agent/options.mdx

```diff
@@ -955,6 +955,24 @@ Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'."
 
 - `bind_addr` Equivalent to the [`-bind` command-line flag](#_bind).
 
+- `cache` Cache configuration of agent. The configurable values are the following:
+
+  - `entry_fetch_max_burst`: The size of the token bucket used to recharge the rate-limit per
+    cache entry. The default value is 2 and means that when cache has not been updated
```

Comment:
The default is now `rate.Inf`.

Reply:
@i0rek The `cache.entry_fetch_max_burst` is actually 2 by default (even if it has no real meaning until `cache.entry_fetch_rate` is set to something different than +Inf...
I propose to add: The default value is 2 if `cache.entry_fetch_rate` is set.

Reply:
I am sorry, I looked at the wrong thing!